src/clinical/service-worker-thread/tasks.ts

```diff
@@ -116,6 +108,20 @@ const sortRecordsByColumn = (
 	return valueSort;
 };
 
+// Sort Invalid Records to Top
+const sortInvalidRecords = (
+	currentRecord: ClinicalInfo,
+	nextRecord: ClinicalInfo,
+	errorIds: Set<number>,
+) => {
+	const currentRecordIsInvalid = errorIds.has(currentRecord.donor_id as number);
+	const nextRecordIsInvalid = errorIds.has(nextRecord.donor_id as number);
```

Comment:
Leaving comment to state I reviewed this type assertion and think its ok in this case. If the donor_id is not a number then the `has` function will fail to match it pretty safely.